### PR TITLE
Upgrade to Ruby 2.7 and PostGIS 12.3.3

### PR DESCRIPTION
**Pull request by dlebauer**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -18,12 +18,12 @@ jobs:
       fail-fast: false
       matrix:
         ruby: 
-          - 2.5
-          - 2.6
-          # - 2.7 is not supported, results in error
+          - 2.7
+        postgres_image:
+          - postgis/postgis:12.3.3
```
Comment (dlebauer) [inline .github/workflows/ci.yml]:
```suggestion
          - postgis/postgis:12-3.3
```

**Comment by robkooper** [comment]
LGTM, can we add some text to the CHANGELOG to tell people that if they use the docker-compose file they will need to dump the database first and reload since this involves a database upgrade.

**Review by robkooper — APPROVED** [review]

**Comment by dlebauer** [comment]
Changelog updated. What are preferred commands to dump and load?

**Comment by robkooper** [comment]
just as an FYI, i have done some testing with posgresql 15 specifically https://hub.docker.com/r/kartoza/postgis/. The nice thing is that this works on the new mac machines and has an ARM container.

**Comment by dlebauer** [comment]
I'll go ahead and merge. FWIW simply adding kartoza/postgres:15-3.3 to the matrix broke the build https://github.com/PecanProject/bety/actions/runs/4556227767/jobs/8036472716

**Pull request merged**